CMakeLists.txt

```diff
@@ -1,81 +1,64 @@
 
-project (Atomic)
+project(Atomic)
 
 cmake_minimum_required(VERSION 2.8.12.1)
 
-set (ATOMIC_SOURCE_DIR ${Atomic_SOURCE_DIR})
-set (CMAKE_MODULE_PATH ${ATOMIC_SOURCE_DIR}/Build/CMake/Modules)
+set(ATOMIC_SOURCE_DIR ${Atomic_SOURCE_DIR})
+set(CMAKE_MODULE_PATH ${ATOMIC_SOURCE_DIR}/Build/CMake/Modules)
 
 include(AtomicGit)
 include(AtomicUtils)
 include(AtomicCommon)
 
 add_definitions(-DATOMIC_ROOT_SOURCE_DIR="${ATOMIC_SOURCE_DIR}" -DATOMIC_ROOT_BUILD_DIR="${CMAKE_BINARY_DIR}")
 
-add_definitions( -DATOMIC_API= -DATOMIC_STATIC_DEFINE -DATOMIC_ATOMIC2D -DATOMIC_LOGGING -DATOMIC_PROFILING)
-
 if (NOT DEFINED ATOMIC_DEV_BUILD)
     set(ATOMIC_DEV_BUILD 1)
-ENDIF()
+ENDIF ()
 
 if (ATOMIC_DEV_BUILD)
     add_definitions("-DATOMIC_DEV_BUILD=1")
-endif()
-
-add_link_libraries_exported (Atomic Box2D Duktape TurboBadger FreeType JO LZ4 PugiXml STB SDL)
-
-if (NOT ATOMIC_BUILD_2D)
-    add_definitions_exported (-DATOMIC_PHYSICS -DATOMIC_3D)
-    add_link_libraries_exported (Bullet StanHull)
```

Comment:
We lost the definition of ```ATOMIC_3D``` here, note ```ATOMIC_BUILD_2D``` used to be used for a purely 2D build, the motivation being to provide a small 2D web player.  We don't strictly need this right now, and seeing that we're building Recast/Detour/Crowd stuff on Web now.  The Web platform isn't currently hooked up to CI, other than when doing binary builds of editor.

```ATOMIC_3D``` does need to be defined, as there are generated guards for the 2D only builds in bindings.